client/src/components/protected/ServerPing.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+import { Button } from "@material-ui/core";
+
+export default ({ ping }) => {
+  //uncomment this for testing individual pings
+  // return <Button variant="outlined" onClick={ping}>ping</Button>;
+
+  //comment this for testing individual pings
+  useEffect(() => {
+    const interval = setInterval(async () => {
+      ping()
+    }, 1000);
+    return () => clearInterval(interval);
+  }, []);
+  return null;
+
+};
```

Comment:
Hey Pavel, I'm not sure I understand how this component would be used. It seems to me that you'd be better off using a regular javascript function